nrfx_mods/README.md

```diff
@@ -0,0 +1 @@
+This directory enables overwriting nrfx source files so we can instead use our own. nrfx is a submodule of this repository, but this location should be checked first, if any issues arise from this area to ensure default functionality hasn't been changed.
```

Comment:
Could we also add a note here that if the nrfx library is updated we should check if the updates should also be applied to the files contain here?